eng/mgmt/mgmtmetadata/batch_data-plane.txt

```diff
@@ -0,0 +1,24 @@
+﻿Installing AutoRest version: 2.0.4283
```

Comment:
What is this?

Reply:
It's a file automatically created by autorest